docs/reference/commandline/plugin_ls.md

```diff
@@ -40,6 +43,35 @@ ID                  NAME                  TAG                 DESCRIPTION
 69553ca1d123        tiborvass/no-remove   latest              A test plugin for Docker   true
 ```
 
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* enabled (boolean - true or false, 0 or 1)
+* capability (string - currently either `volumedriver` or `authz`)
+
+### enabled
+
+The `enabled` filter matches on plugins enabled or disabled.
+
+### capability
+
+The `capability` filter matches on plugin capabilitiess. One plugin
+might have multiple capabilities. Currently `volumedriver` and `authz`
```

Comment:
@thaJeztah @vieux I think both `network` driver and `ipam` driver are supported (though they are not documented). 

Also legacy plugin supports `graph` driver. Not sure if v2 plugin support graph driver or not.